### PR TITLE
Changed role on CompanyRole to nullable

### PR DESCRIPTION
**Pull request by dowmeister**
This PR fixes the CompanyRole when API returns the `order` property at null

### CODE REVIEW

File: src/Models/CompanyRole.php [hunk 1]
```diff
@@ -62,7 +62,7 @@ public function __construct(Client $client, array $role)
 
         $this->id = $this->getValue('id');
         $this->name = $this->getValue('name');
-        $this->order = $this->getValue('order');
+        $this->order = $this->getValue('order', null);
```
Comment (ShawnCZek) [inline src/Models/CompanyRole.php]: The second parameter is `null` by default. This implicit value is unnecessary and should be removed.

**Review by ShawnCZek — CHANGES_REQUESTED** [review]
May you also please share an example payload where the order is actually `null`? I have just checked the official documentation and it states that the field is required.

**Comment by dowmeister** [comment]
My VTC (id 24 - https://api.truckersmp.com/v2/vtc/24/roles) were returning null for Owner role yesterday but after reording roles from the site, the null turned to 0.

**Comment by ShawnCZek** [comment]
I have forwarded this to web developers as this sounds more like an issue with the API, especially if it is fine now, but it was not before. I, or someone else, will get back to you once this has been internally discussed.

**Comment by ShawnCZek** [comment]
This was a bug that was already fixed four years ago. However, it still affected companies created before that date. Therefore, we ran a query to fix all these leftover null values, and the API should no longer return `null` for the role order.

Thank you for bringing this to our attention. 🎉

**Pull request closed without merging**